client/lib/domains/is-dot-gay-notice-required.js

```diff
@@ -0,0 +1,7 @@
+import { find, get } from 'lodash';
+
+export function isDotGayNoticeRequired( productSlug, productsList ) {
+	const product = find( productsList, [ 'product_slug', productSlug ] ) || {};
```

Comment:
Same comment here; we don't really need lodash I think.

```suggestion
	const product =  productList.find( product => product.product_slug === productSlug ) ?? {};
```